.github/stale.yml

```diff
@@ -0,0 +1,42 @@
+# Stale config for https://github.com/marketplace/stale
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - security
+  - blocked
+
+# Label to use when marking a pull request as stale
+staleLabel: stale
+
+pulls:
+  # Number of days of inactivity before a pull request becomes stale
+  daysUntilStale: 7
+
+  # Number of days of inactivity before a stale pull request is closed
```

Comment:
Again, is the goal pruning or surfacing?  With PRs, closing might help with surfacing more than with issues, since presumably the PR author will have some incentive to reopen.  If the goal is pruning, none of that matters, of course.

Reply:
the goal in the near term is resurface, but I think there are some PRs that need pruning if resurfacing doesn't drive them forward